examples/components/PNEUMONIA/traininsilo/run.py

```diff
@@ -353,16 +355,18 @@ def test(self):
         self.model_.eval()
         test_loss = 0
         correct = 0
+        num_sample = 0
         with torch.no_grad():
             for data, target in self.test_loader_:
                 data, target = data.to(self.device_), target.to(self.device_)
+                num_sample += data.shape[0]
                 output = self.model_(data)
                 test_loss += self.loss_(output, target).item()
                 pred = output.argmax(dim=1, keepdim=True)
                 correct += pred.eq(target.view_as(pred)).sum().item()
 
-        test_loss /= len(self.test_loader_.dataset)
-        acc = correct / len(self.test_loader_.dataset)
+        test_loss /= num_sample
```

Comment:
afaik, by default the loss functions in PyTorch calculate mean loss across samples in batch, thus I believe correct way to calculate average loss is to divide this by number of batches